sklearn/_min_dependencies.py

```diff
@@ -29,7 +29,7 @@
     "scipy": (SCIPY_MIN_VERSION, "build, install"),
     "joblib": (JOBLIB_MIN_VERSION, "install"),
     "threadpoolctl": (THREADPOOLCTL_MIN_VERSION, "install"),
-    "cython": (CYTHON_MIN_VERSION, "build"),
+    "Cython": (CYTHON_MIN_VERSION, "build"),
```

Comment:
Question: is the casing impacting a build step? Is this related to the use of `pyproject.toml` / `setuptools.build_meta`?

Reply:
Pushing the pyproject.toml commit was a mistake.

Reply:
As for the question, the case is not required. I had a strict check to make sure `pyproject.toml` was consistent with `_min_dependencies.py` which was case-sensitive.